### ISSUE TITLE
Hyperparmeters for BN layer

### ISSUE THREAD

**Issue by developer0hye** [ultralytics/yolov5#7165]
https://github.com/ultralytics/yolov5/blob/6134ec5d9484ac9ac743121b1c74709e93c68a88/utils/torch_utils.py#L158-L159

Hi @glenn-jocher !

I have a question.

How are these values set?

`eps` is much higher than its default value(1e-5), and `momentum` is lower than its default value(1e-1).

**Comment by glenn-jocher** [comment]
@developer0hye hi there!

The `eps` and `momentum` hyperparameters for the BatchNorm layer are set empirically and have been found to work well during experimentation. These values have been adjusted to better suit the YOLOv5 architecture. Feel free to experiment with different values based on your specific use case, and let us know if you find improvements.

You can explore further insights on how the YOLOv5 model and its components are implemented in our documentation at https://docs.ultralytics.com/yolov5/.

Cheers!
